WordPress/src/main/java/org/wordpress/android/util/WPActivityUtils.java

```diff
@@ -15,4 +16,12 @@ public static Context getThemedContext(Context context) {
         }
         return context;
     }
+
+    public static String getNonNullString(TextView textView) {
+        if (textView != null && textView.getText().length() != 0) {
```

Comment:
[This discussion](http://stackoverflow.com/questions/19590472/does-edittext-gettext-ever-returns-null#answers) mentions that getText will never return null so it should be safe. I think we can safely get rid of the length check too.